client/packages/common/src/intl/locales/fr/replenishment.json

```diff
@@ -35,9 +35,9 @@
   "label.hide-stock-over-minimum": "Masquer articles supérieurs au minimum",
   "label.location": "Emplacement",
   "label.max": "Max",
-  "label.max-months-of-stock": "MDS Maximum",
```

Comment:
These are again best guesses at translations. (not reviewed by a native speaker)